src/contracts/Staking.sol

```diff
@@ -92,10 +128,16 @@ contract Staking is Ownable {
         stakingToken = _stakingToken;
         require(_rewardToken != address(0));
         rewardToken = _rewardToken;
+        require(_rewardToken != address(0));
```

Comment:
would recommend grouping all requires checks as the first thing we do in the constructor, while maybe trivial it will revert before we waste gas on anything else.